addons/main/functions/fnc_receiveDamage.sqf

```diff
@@ -83,10 +84,16 @@ if (_newHP isEqualTo 0) exitWith {
         };
     } else {
         // damage to downed units
-        private _downedHp = _unit getVariable [QGVAR(downedHp), _maxHp];
+        private _downedHits = 0;
+        If (_downDamage > 1) then {
+            _downedHits = ((_unit getVariable [QGVAR(downedHits),0]) + 1);
+            _unit setVariable [QGVAR(downedHits), _downedHits];
+        };
+        if (_downDamage == 2 && {_downedHits < GVAR(downedDamageHits)}) exitWith {};
+        private _downedHp = _unit getVariable [QGVAR(downedHp), (_maxHp * GVAR(downedDamageHP))];
         private _newDownedHP = (_downedHp - _damage) max 0;
         _unit setVariable [QGVAR(downedHp), _newDownedHP];
-        if (_downedHp isEqualTo 0) then {
+        if (_downedHits >= GVAR(downedDamageHits) || {_downedHp isEqualTo 0}) then {
```

Comment:
```suggestion
        if (_downedHits >= GVAR(downedDamageHits) || {_newDownedHP isEqualTo 0}) then {
```
Looks like I left a bug in there from the earlier feature. it checks the old downed HP and not the new value, meaning it would have killed you after the next bullet.